scripts/utils/bdlClassnameManager

```diff
@@ -0,0 +1,225 @@
+                .map(converter)
+                .sort(lengthSorter) // sort order matters here for regex
+                .join('|'), 'g'
+        );
+    } else if (fileType === 'js') {
+        // The majority of .js classnames are surrounded by quotation marks
+        // Restrict to those prefixes to avoid overrwriting variable names
+        const converter = entry => jsPrefixes.map(prefix => `${prefix}${entry}`).join('|');
+        reBadName = new RegExp(Object.keys(currentConversionMap).map(converter).join('|'), 'g');
+    } else {
+        console.error(colors.yellow('Unrecognized file type for this tool. skipping', fileName, '...'));
+        process.exit(0);
+    }
+
+    try {
+        let fileToParse = await readFile(fileName, { encoding: 'utf8' });
+
+        switch (operation) {
+            case 'check':
+                // determine if the file is using one of the legacy variable names
+                const foundMatch = reBadName.test(fileToParse);
+                if (foundMatch) {
+                    // Exit with an error code: we should not find bad names
+                    if (verboseMode) {
+                        console.error(colors.red('Bad BDL class name found in', fileName));
+                    }
+                    process.exit(1);
+                }
+
+                break;
+            case 'append': {
+                const matches = [...new Set(fileToParse.match(reBadName))];
+
+                if (verboseMode) {
+                    printVerboseOperationMessage(fileName, matches);
+                }
+
+                matches
+                    .sort(lengthSorter) // sort prevents root classnames ('btn') from preemptively changing longer classnames ('btn-primary')
+                    .forEach((match) => {
+                        if (fileType === 'scss') {
+                            fileToParse = fileToParse.replace(new RegExp(`\\${match}(?!-)`, 'g'), `${scssPrefix}${currentConversionMap[kebabCase(match)]}`);
+                        } else if (fileType === 'js') {
+                            const quotationMark = match[0];
+                            if (jsPrefixes.includes(quotationMark)) {
+                                fileToParse = fileToParse.replace(new RegExp(`(?<!-)${match}(?!-)`, 'g'), `${match} ${currentConversionMap[kebabCase(match)]}`);
+                            } else if (verboseMode) {
+                                console.warn(colors.yellow('Incorrect match for js files. skipping', fileName, '...'));
+                            }
+                        }
+                    });
+
+                handleWriteFile(matches, fileName, fileToParse, verboseMode);
+
+                break;
+            }
+            case 'extend': {
+                const matches = [...new Set(fileToParse.match(reBadName))];
+
+                if (verboseMode) {
+                    printVerboseOperationMessage(fileName, matches);
+                }
+
+                matches
```

Comment:
You may want to throw in a few lines of whitespace and some recognizable heading to visually separate the two sections of the stylesheet. It may make it easier to clean up the file later.